site/docs/configuration/expected-outputs/deterministic.md

```diff
@@ -261,6 +262,72 @@ assert:
     value: file://./path/to/schema.json
 ```
 
+### Is-SQL
+
+The `is-sql` assertion checks if the LLM output is a valid SQL statement.
+
+Example:
+
+```yaml
+assert:
+  - type: is-sql
+```
+
+You can optionally set a `databaseType` in the `value` to determine the specific database syntax that your LLM output will be validated against. The default database syntax is MySQL. For a complete and up-to-date list of supported database syntaxes, please refer to the [node-sql-parser documentation](https://github.com/taozhi8833998/node-sql-parser?tab=readme-ov-file#supported-database-sql-syntax).  
+The supported database syntax list:
+
+- Athena
+- BigQuery
+- DB2
+- FlinkSQL
+- Hive
+- MariaDB
+- MySQL
+- Noql
+- PostgresQL
+- Redshift
+- Snowflake(alpha)
+- Sqlite
+- TransactSQL
+
+Example:
+
+```yaml
+assert:
+  - type: is-sql
+    value:
+      databaseType: 'MySQL'
+```
+
+You can also optionally set a `whiteTableList`/`whiteColumnList` in the `value` to determine the SQL authority list that your LLM output will be validated against.  
+The format of whiteTableList:
+
+```md
```

Comment:
nit: this code block and the one below are not markdown, so you can remove the `md` language

Reply:
Should I leave it as plain text? This is what plain text looks like:
![7c4acddefa2ac9bbdabcce3bc27a055](https://github.com/promptfoo/promptfoo/assets/113162439/6cbed93c-529d-4c55-9cc4-43df1b10e73e)

Reply:
got it